src/tests/createHeadlessForm.test.js

```diff
@@ -1035,6 +1035,62 @@ describe('createHeadlessForm', () => {
       );
     });
```

Comment:
I'll review this today.

Reply:
Ready for a new review

Reply:
on it now 👀

Reply:
Review done! Left a few comments here and in the internal docs, and this is ready to go!